### ISSUE TITLE
Auto Inject Issue

### ISSUE THREAD

**Issue by 34736384** [master131/ExtremeInjector#49]
All the injection method has a chance where it tries to inject way too early, before kernel32.dll loads in, which causes it to fail. Sometimes it auto injects fine but sometimes it just says can't load dependency from kernel32.dll

**Comment by master131** [comment]
Adjust the injection delay in the settings.

**Comment by 34736384** [comment]
Yes I do know about the injection delay, but in my case I would need to have the dll loaded before the program initializes

**Comment by master131** [comment]
kernel32.dll is a core system DLL, its pretty unlikely for it to be missing - are you sure whatever you are injecting into doesn't have anti-cheat protection enabled or something.

**Comment by 34736384** [comment]
Yes, I am sure that the process I'm injecting into does not have any anti-cheat protection. Auto-inject works, but has a chance to fail.

**Comment by pythonmcpi** [comment]
Try manual injection then?

**Comment by 34736384** [comment]
> Try manual injection then?

The reason I need auto inject is to able to hook before the program initializes it's security routine
